test/demo.css

```diff
@@ -9,10 +9,12 @@ body {
 	font-family: Figtree, sans-serif;
 	width: 375px;
 	height: auto;
-	min-height: 100vh;
+	min-height: 812px;
```

Comment:
Better use calc() if you want to keep footer visible, otherwise should not be a fixed value

Reply:
I have set to min 812 because the figma screen preview are with using exactly 812 height and 375 width. In develop phase it can help to look the same. I would say just I keep like that and we can modify later.